cibuildwheel/__main__.py

```diff
@@ -206,6 +212,9 @@ def main() -> None:
     build_verbosity_str = get_option_from_environment(
         "CIBW_BUILD_VERBOSITY", platform=platform, default=""
     )
+    prerelease_pythons = args.prerelease_pythons or cibuildwheel.util.strtobool(
+        get_option_from_environment("CIBW_PRERELEASE_PYTHONS", platform=platform, default="0")
```

Comment:
Grrr, this will either require it be added to the pyproject.toml (yuck, not at all a good idea for a flag like this), or removing the platform specific choices, or adding custom code back to handle platform choices but no pyproject.toml support for #684. Does it have to support platform specific variants?

Reply:
I guess no platform specific variant is not an issue for this one (and shall definitely not be in `pyproject.toml`)

Reply:
I hadn't fully considered the pyproject.toml angle, but I think you're right, it would be kinda crazy to write this into a `pyproject.toml`! I could imagine that perhaps this option would only be specifiable at runtime from flag or env var, if that makes you feel better henryiii? I could imagine in the docs something like-

> Only configurable at runtime via `--prerelease-pythons` or `CIBW_PRERELEASE_PYTHONS`. Cannot be set in `pyproject.toml`.

It's interesting that `CIBW_OUTPUT_DIR` is another one of these runtime-only options? (maybe you've discussed this in #547, apologies, I'm behind!)

I'm not so worried about the platform-specific thing here, it seems pretty harmless.

Reply:
Okay, I'll remove the variants for now.

> `CIBW_OUTPUT_DIR`

This is undocumented currently, actually. It's discussed in #547, though it's a bit of an open question. I had it in the toml then I took it out, but could put it back in.